x/projects/keeper/migrations.go

```diff
@@ -25,3 +27,71 @@ func (m Migrator) Migrate2to3(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+// Migrate3to4 implements protobuf migration from v3 to v4:
+// Now the project protobuf has admin and subscription policies
+func (m Migrator) Migrate3to4(ctx sdk.Context) error {
```

Comment:
Same comments as in `x/plans/`:
1. The migration from 2 to 3 didn't happen yet, so let's combine this into the exiting/upcoming Migrate2to3().
2. We decided to put actual migration code in v3/... subdir if possible. In this case, following x/bank/... example, introduce x/plans/migrations/v3/store.go:MigrateStore(...) and call it from here. Because the migration code needs access to keeper.plansFS, maybe we can add an API in x/plans/keeper/migrations.go to expose it, with a clear comment stating the API should only be used in migrations.

Reply:
done